docs/pages/database-access/guides/snowflake.mdx

```diff
@@ -173,11 +171,11 @@ To retrieve credentials for a database and connect to it:
 $ tsh db connect example-snowflake
 ```
 
-You can optionally specify the user to use by default
-when connecting to the database instance:
+You can optionally specify the database user and the database name to use by
+default when connecting to the database instance:
```

Comment:
Are these edits not in the original PR?

Reply:
they only went to `master` and were not backported.

Reply:
I see, these changes were in #14898 but not included in its backport.